xml/admin_gui_dashboard.xml

```diff
@@ -2199,6 +2199,94 @@ scrape_configs:
+      </para>
+    </sect3>
+    <sect3 xml:id="dash.rbd_qos.pool_create">
+      <title>Configuring options on a new pool</title>
+      <para>
+        To create a new pool and configure RBD configuration options on it, click <guimenu>Pools</guimenu>, then click <guimenu>Create</guimenu>.  Select <guimenu>replicated</guimenu> as "Pool type".  You will then need to add the <guimenu>rbd</guimenu> application tag to the pool, to be able to configure the RBD QoS options.
+      </para>
+      <note>
+        <para>
+          It is not possible to configure RBD QoS configuration options on an erasure coded pool!  To configure the RBD QoS options for erasure coded pools, you will have to edit the replicated metadata pool of an RBD image.  The configuration will then be applied to the erasure coded data pool of that image.
+        </para>
+      </note>
+    </sect3>
+    <sect3 xml:id="dash.rbd_qos.pool_edit">
+      <title>Configuring options on an existing pool</title>
+      <para>
+        To configure RBD QoS options on an existing pool, click <guimenu>Pools</guimenu>, then click the pools' row in the table and click <guimenu>Edit</guimenu> at the top left of the table to open the edit dialog.
+      </para>
+      <para>
+        You should see the "RBD Configuration" Section in the dialog, followed by a <guimenu>Quality of Service</guimenu> section. 
+      </para>
+      <note>
+        <para>
+          If you neither see the "RBD Configuration", nor the "Quality of Service" section, you are likely either editing an *erasure coded* pool, which cannot be used to set RBD configuration options, or the pool isn't configured to be used by RBD images.  In case of the latter, click the <guimenu>pencil</guimenu> to the right of the "Applications" label to add the <guimenu>rbd</guimenu> application tag to the pool.  Immediately afterwards, the corresponding configuration sections should show up.
+        </para>
+      </note>
+    </sect3>
+    <sect3 xml:id="dash.rbd_qos.qos">
+      <title>Configuration Options</title>
+      <para>
+        Click "Quality of Service +" to expand the configuration options.  A list of all available options will show up.  The units of the configuration options are already shown in the input fields.  In case of any bytes per second (BPS) option, you are free to use values like "1M" or "5G".  They will automatically be converted to "1 MB/s" and "5 GB/s" respectively.
```

Comment:
```suggestion
        Click <guimenu>Quality of Service +</guimenu> to expand the configuration options.  A list of all available options will show up.  The units of the configuration options are already shown in the input fields.  In case of any bytes per second (BPS) option, you are free to use shortcuts such as '1M' or '5G'.  They will be automatically converted to '1 MB/s' and '5 GB/s' respectively.
```